mailpoet/views/upgrade.html

```diff
@@ -224,32 +220,43 @@ <h1 class="mailpoet-h0">
+      </h3>
+      <h3>
+        <%= _x('This plan includes:', 'This text resides in the Upgrade page: /wp-admin/admin.php?page=mailpoet-upgrade') %>
+      </h3>
       <ul class="mailpoet-premium-page-bullet-list">
-        <li><%= _x('Quickly deploy an easy to learn email solution for all your clients', 'This text resides in the Premium page: /wp-admin/admin.php?page=mailpoet-premium') %></li>
-        <li><%= _x('Easily manage all your customers from a single account', 'This text resides in the Premium page: /wp-admin/admin.php?page=mailpoet-premium') %></li>
-        <li><%= _x('Activate MailPoet Premium features on an unlimited number of websites', 'This text resides in the Premium page: /wp-admin/admin.php?page=mailpoet-premium') %></li>
-        <li><%= _x('Send unlimited emails for each site up to 1,000 subscribers', 'This text resides in the Premium page: /wp-admin/admin.php?page=mailpoet-premium') %></li>
-        <li><%= _x('Send to more than 1,000 subscribers with any other sending methods (your host, Sendgrid & Amazon SES)', 'This text resides in the Premium page: /wp-admin/admin.php?page=mailpoet-premium') %></li>
-        <li><%= _x('Exchange with our deliverability manager for any sending issues', 'This text resides in the Premium page: /wp-admin/admin.php?page=mailpoet-premium') %></li>
+        <li><%= _x('Use MailPoet on 50 websites', 'This text resides in the Upgrade page: /wp-admin/admin.php?page=mailpoet-upgrade') %></li>
+        <li><%= _x('Send unlimited emails', 'This text resides in the Upgrade page: /wp-admin/admin.php?page=mailpoet-upgrade') %></li>
+        <li><%= _x('Without MailPoet branding', 'This text resides in the Upgrade page: /wp-admin/admin.php?page=mailpoet-upgrade') %></li>
+        <li><%= _x('Priority customer support', 'This text resides in the Upgrade page: /wp-admin/admin.php?page=mailpoet-upgrade') %></li>
+      </ul>
+      <h3>
+        <%= _x('All Starter features, plus:
+        ', 'This text resides in the Upgrade page: /wp-admin/admin.php?page=mailpoet-upgrade') %>
+      </h3>
+      <ul class="mailpoet-premium-page-bullet-list">
+        <li><%= _x('Detailed engagement statistics', 'This text resides in the Upgrade page: /wp-admin/admin.php?page=mailpoet-upgrade') %></li>
+        <li><%= _x('Subscriber segmentation', 'This text resides in the Upgrade page: /wp-admin/admin.php?page=mailpoet-upgrade') %></li>
+        <li><%= _x('Google Analytics integration', 'This text resides in the Upgrade page: /wp-admin/admin.php?page=mailpoet-upgrade') %></li>
       </ul>
       <a
         target="_blank"
-        href="https://www.mailpoet.com/premium/agency"
+        href="<%= add_referral_id("https://account.mailpoet.com/?s=" ~ subscriber_count ~ "&g=agency&billing=monthly&email=" ~ current_wp_user.user_email ~ '&utm_source=plugin&utm_medium=premium&utm_campaign=purchase') | escape('html_attr') %>"
```

Comment:
Does it need URL escaping? I guess count doesn't and email... shouldn't contain any unsafe characters either?

Reply:
@costasovo I meant that I don't see URL-escaping. There is `escape('html_attr')` but that's something else. The variables are escaped for HTML attribute but not for URL. But I guess in this case it's safe.

Reply:
Ahh sry. Now I understand. All those inputs should be safe I guess.